python/private/pypi/extension.bzl

```diff
@@ -788,6 +810,12 @@ The CPU architecture name to be used.
 :::{note}
 Either this or {attr}`env` `platform_machine` key should be specified.
 :::
+""",
+    ),
+    "constraint_values": attr.label_list(
```

Comment:
Followup thought for subsequent PR: replace constraint_values and target_settings with simply "config_settings" ?

The custom platform stuff in the toolchains has target_compatible_with (constraints) and target_settings because that's what the toolchain() rule accepts. However, for the pip stuff, it just needs something valid for a select() key.

Reply:
Correct, I like this suggestion. `config_settings` does sound more natural.